action.yaml

```diff
@@ -21,6 +13,16 @@ inputs:
         1. set this input equal to the `mount path` of needed volume,
         2. add its read-write reference to the `inputs.volumes` list
       Docs: https://mlflow.org/docs/latest/tracking.html#artifact-stores
+  backend_store_uri:
+    default: ""
+    descr: |
+      URI to the storage, which should be used to dump experiments, their metrics, registered models, etc.
+      Examples:
+        Empty string: in this case the model registry functionality will not work
+        Postgress DB: postgresql://postgres:password@job-1231-123123-123.platform-jobs:5432
+        SQLite DB: sqlite:///some/path/mlflow.db - you cannot use the platform storage if it's running in Azure, but could use disk
```

Comment:
`if it's running in Azure` - while this is true, I doubt regular platform users are going to know where the platform is running. How about creating a task to add a check for this scenario in the future? 
No need to do this now (seems low-priority), but in general we should warn our users IF the platform is running in Azure, not tell them "go and check where the platform is running".

Reply:
added in 
8e40190